projects/packages/jetpack-mu-wpcom/src/features/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-launchpad-navigator.php

```diff
@@ -47,6 +47,11 @@ public function register_routes() {
 							'type'              => array( 'null', 'string' ),
 							'validate_callback' => array( $this, 'validate_checklist_slug_param' ),
 						),
+						'remove_checklist_slug' => array(
+							'description' => 'The slug of the checklist to set as active.',
```

Comment:
This comment is not accurate.
```suggestion
							'description' => 'The slug of the checklist to remove from the active list.',
```